src/components/media-loader.js

```diff
@@ -257,13 +259,38 @@ AFRAME.registerComponent("media-loader", {
     }
   },
 
+  refresh() {
+    if (this.networkedEl && !NAF.utils.isMine(this.networkedEl) && !NAF.utils.takeOwnership(this.networkedEl)) return;
```

Comment:
Hmm, if you hit refresh before `networkEl` existed I think you would end up not exiting early and would bump the version. May be fine but a bit weird.

Reply:
this exits early if you call `refresh()` before `networkedEl`, no?